lib/checksum-tools/remote.rb

```diff
@@ -28,15 +28,15 @@ def initialize(host, user, *args)
         write_remote_properties
       end
     end
-    
+
     def openssl
-      remote_properties[:openssl]
+      remote_properties[:openssl] || remote_properties['openssl']
     end
-    
+
     def sftp
       @sftp ||= begin
-        auth_methods = %w(gssapi-with-mic publickey hostbased) if defined? Net::SSH::Kerberos
-        auth_methods ||= %w(publickey hostbased)
+        auth_methods = %w(publickey hostbased keyboard-interactive) if defined? Net::SSH::Kerberos
+        auth_methods ||= %w(publickey hostbased keyboard-interactive)
```

Comment:
Heh, yes, yes they are.  I even thought about that, but then neglected to fix it.  I'll pull it.